addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/ScEnvironmentCard.js

```diff
@@ -57,6 +57,9 @@ class ScEnvironmentCard extends React.Component {
   getConfiguration(envTypeConfigId) {
     const configsStore = this.getEnvTypeConfigsStore();
     const config = configsStore.getEnvTypeConfig(envTypeConfigId);
+    if (config === undefined) {
```

Comment:
This assumes this method will only be used for returning the `name` and `instanceType` attributes. Could you rather move the check for `undefined` in the ScEnvironmentCard component?

Reply:
Good point. Moving it now.